doc/source/user_guide/indexing.rst

```diff
@@ -1126,7 +1126,81 @@ Mask
 
 .. _indexing.query:
 
-The :meth:`~pandas.DataFrame.query` Method
+The ``Series`` :meth:`~pandas.Series.query` Method
```

Comment:
i wouldn't add any of this, just make a small mention in the next section.

Reply:
no problem, the next section as in "The :meth:`~pandas.DataFrame.query` Method"? and write up a few words there to mention Series.query() method?